extras/kms/Makefile

```diff
@@ -0,0 +1,27 @@
+
+
```

Comment:
honestly, I'm not a dev that's bothered by extra while space.  If there's a std formatter available and we can enforce style like this via tooling then I'm all for it... otherwise I sort of personally gave up decades ago worry about bits like this.... now if there are formatters like `fmt`, `gofumpt`, `goimports` which a team chooses to align on, which we can automate, then of course I'm very happy to oblige.